config/cable.yml

```diff
@@ -5,6 +5,4 @@ test:
   adapter: test
 
 production:
-  adapter: redis
-  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
-  channel_prefix: dradispro_production
+  adapter: async
```

Comment:
are we better off w/ 2 different files than with a `defined?(Dradis::Pro)` conditional?

Would :async blow up if we provide :url and :channel_preffix? If not, and we can getaway with a single line conditional for the adapter, we may be better off than keeping two different versions of the file in sync